src/discussions/learners/LearnersContentView.jsx

```diff
@@ -0,0 +1,128 @@
+  selectLearnerAvatar,
+  selectPostsOfUser, selectResponsesOfUser,
+} from './data/selectors';
+import { fetchUserComments, fetchUserPosts } from './data/thunks';
+import messages from './messages';
+
+export const TABS = {
+  POSTS: 'posts',
+  COMMENTS: 'comments',
+  RESPONSES: 'responses',
+};
+
+function LearnersContentView({ intl }) {
+  const [tab, setTab] = useState(TABS.POSTS);
+  const { courseId, learnerUsername: userId } = useParams();
+  const dispatch = useDispatch();
+  const comments = useSelector(selectCommentsOfUser(userId));
+  const posts = useSelector(selectPostsOfUser(userId));
+  const responses = useSelector(selectResponsesOfUser(userId));
+  const apiStatus = useSelector(learnersLoadingStatus());
+  const learner = useSelector(selectLearner(userId));
+  const avatar = useSelector(selectLearnerAvatar(userId));
+
+  useEffect(() => {
+    dispatch(fetchUserPosts(courseId, userId));
+    dispatch(fetchUserComments(courseId, userId));
+  }, [courseId, userId, tab]);
+
+  const handleTabChange = (val) => setTab(val);
+
+  return (
+    <div className="learner-content d-flex flex-column">
+      <Card>
+        <Card.Body>
+          <div className="d-flex flex-row align-items-center m-3">
+            <Avatar src={avatar} alt={userId} />
+            <span className="font-weight-bold mx-3">
+              {learner.name}
+            </span>
+            <div className="ml-auto">
+              <IconButton iconAs={Icon} src={MoreHoriz} alt="Options" />
+            </div>
+          </div>
+        </Card.Body>
+        <Card.Footer className="pb-0 bg-light-200">
+          <div className="d-flex flex-row w-100 my-2">
+            <div className="spacer w-25" />
+            <div className="w-50 text-center">
+              <ButtonGroup>
+                <Button data-testid={`${TABS.POSTS}-btn`} variant="primary" className={tab !== TABS.POSTS ? 'bg-white border-gray-100 text-primary' : ''} onClick={() => handleTabChange(TABS.POSTS)}>
+                  {intl.formatMessage(messages.postsTab)} <span className="ml-3">{posts.length}</span>
+                  <span className="ml-3">
+                    <Icon src={Report} />
+                  </span>
+                </Button>
+                <Button data-testid={`${TABS.RESPONSES}-btn`} variant="primary" className={tab !== TABS.RESPONSES ? 'bg-white border-gray-100 text-primary' : ''} onClick={() => handleTabChange(TABS.RESPONSES)}>
+                  {intl.formatMessage(messages.responsesTab)} <span className="ml-3">{responses.length}</span>
+                </Button>
+                <Button data-testid={`${TABS.COMMENTS}-btn`} variant="primary" className={tab !== TABS.COMMENTS ? 'bg-white border-gray-100 text-primary' : ''} onClick={() => handleTabChange(TABS.COMMENTS)}>
+                  {intl.formatMessage(messages.commentsTab)} <span className="ml-3">{comments.length}</span>
+                </Button>
+              </ButtonGroup>
```

Comment:
I think there is a Tabs component in paragon for this.

Reply:
@xitij2000 The Tabs component in the Paragon library doesn't provide the same flexibility.
1. The Tabs and the content couldn't be defined separately to get the same UI as in the mockups.
2. Adding custom elements like the Report Icon in the tab switches isn't possible either.
So, I resorted to implementing it using the button groups.